op-challenger/game/fault/types/position.go

```diff
@@ -55,11 +70,19 @@ func boolToInt(b bool) int {
 	}
 }
 
+func (p Position) parentIndexAtDepth() *big.Int {
+	return big.NewInt(0).Div(p.IndexAtDepth(), big.NewInt(2))
+}
+
+func (p Position) DefendsParent(parentIndex *big.Int) bool {
+	return p.parentIndexAtDepth().Cmp(parentIndex) != 0
```

Comment:
A position can't defend or attack its parent - claims attack or defend their parent by moving to a child or a different subtree. We're really asking if this position is to the right of the supplied position.

I'd also say we should pass the whole `Position` as argument rather than exposing the implementation details of it using index at depth.  So we'd get an api of:

```go
func (p Position) RightOf(pos Position) bool {
```

Reply:
So this is pulling from the old `func (c *Claim) DefendsParent() bool {`. I think that implementing `(*Claim) DefendsParent` in terms of `(Position) RightOf` would work & make the most sense.